README.md

```diff
@@ -8,3 +8,11 @@ Quote of the day
 The difference between stupidity and genius is that genius has its limits.
 *Albert Einstein*
 
+Message to Martin:
+------------------
+
+Hi Martin - how are you doing?
+I hope you, Anne and the boys are doing well!
+Are you still in the same place in Hout Bay?
```

Comment:
Very well. Yes, still the same place. We like it there so it would take quite a bit to uproot.